src/modules/users/components/TokenActivation/TokenActivationContent/TokenActivationContent.tsx

```diff
@@ -2,25 +2,40 @@ import React, { useState } from 'react';
 import { FormattedMessage, defineMessages } from 'react-intl';
 
 import { Tab, Tabs, TabList, TabPanel } from '~core/Tabs';
+import { useClaimableStakedMotionsQuery } from '~data/generated';
+
+import TokensTab, { TokensTabProps } from '../TokensTab';
+import StakesTab from '../StakesTab/StakesTab';
 
-import TokensTab, { TokensTabProps } from './TokensTab';
-import StakesTab from './StakesTab';
 import styles from './TokenActivationContent.css';
 
 const MSG = defineMessages({
   yourTokens: {
-    id: 'users.TokenActivation.TokenActivationContent.yourTokens',
+    id: 'TokenActivation.TokenActivationContent.yourTokens',
```

Comment:
My bad, I think I explained myself badly. I wasn't saying that the ids shouldn't have the `users.`, just that the `TokenActivationContent` should've been removed from the ids in  `StakesTab` and `StakesListItem` (And any other component that is not used exclusively in the `TokenActivationContent`), but the `users.` in the ids is correct so long as they are inside the `users` module

Reply:
@ArmandoGraterol Whats the guidance on MSG ids?
I checked other instances of ids and the string.paths used. Of course they need to be unique, and follow the dir hierarchy. (And should go back far enough for uniqueness)
But different people go back differing amounts. @rdig goes back to `core` but most others start from the first dir after `components`.

Is there a reason to go back as far as `users`? Other than personal choice? (just for me to understand and decide on a rule that i can apply consistently.)

Reply:
Generally speaking, the criteria that are normally used (or at least the one that I use always) is to see where is the component going to be used.

So, if you take a look at the components that we have in the `core` module, most of them start with just the name of the component, they don't go as far as to include `core.` into the id and that's because the core components normally are **NOT** used, at least exclusively, inside the `core` module, they are used all over the dapp. 

Now, if you go to look at some components at the `dashboard` module, they most likely should begin with a `dashboard.` and that's because those components are exclusively used inside that module, they don't go outside it.

In summary, the reason why I would expect these ids you added/updated, to begin with, `users.` is because those components are just being used inside the `users` module and the id should reflect that. If they don't, it's assumed that these components are being used in other modules other than `users`.